systemtest/approvals/TestOTLPGRPCLogs.approved.json

```diff
@@ -39,6 +39,9 @@
                 },
                 "name": "unknown"
             },
+            "session": {
+              "id": ""
```

Comment:
We now always get a session object. We could condition that on ID being set, but it would require going through an apm-data PR again.

Reply:
IMO this is fine.